htstress.c

```diff
@@ -140,10 +147,16 @@ static void init_conn(int efd, struct econn *ec)
         exit(1);
     }
 
+    // manipulate fd, F_SETFL: set file status flags,
```

Comment:
Always use C-style comments. That is, `/*` and `*/`.

Reply:
fixed in https://github.com/sysprog21/khttpd/pull/8/commits/be11e24a0e094e21f849d5e28b393a9f9df55cfc